_posts/en/newsletters/2024-03-27-newsletter.md

```diff
@@ -162,7 +162,46 @@ answers posted since our last update.*
 nswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
-FIXME:bitschmidty
+- [What are the risks of running a pre-SegWit node (0.12.1)?]({{bse}}122211)
+  Michael Folkson, Vojtěch Strnad, and Murch list downsides to running Bitcoin
+  Core 0.12.1 for an individual user including higher risk of accepting an
+  invalid transaction or block, increased vulnerability to double spend attacks,
+  higher reliance on others to do updated consensus validation, much slower
+  block validation, missing many performance improvements, inability to use
+  [compact block relay][topic compact block relay], not relaying ~95% of current
+  unconfirmed transactions, less accurate [fee estimation][topic fee
+  estimation], and vulnerability to security issues fixed in previous versions.
+  Wallet users of 0.12.1 would also miss out in developments around
+  [miniscript][topic miniscript], [descriptor][topic descriptors] wallets, and
+  the fee savings and additional script capabilities enabled by [segwit][topic
+  segwit], [taproot][topic taproot], and [schnorr signatures][topic schnorr
+  signatures]. Effects on the Bitcoin network if Bitcoin Core
+  0.12.0 were more broadly adopted could include: higher chance of invalid
```

Comment:
```suggestion
  0.12.0 was more broadly adopted could include a higher chance of invalid
```

Reply:
IMO was fine before.

Reply:
What is being more broadly adopted is Bitcoin Core 0.12.0, which is singular. This word I'm suggesting (were, was) applies to "Bitcoin Core 0.12.0", not to "effects". So "was" is the correct word to use here.